pilot/adapter/config/crd/generate.sh

```diff
@@ -47,7 +47,7 @@ var knownTypes = map[string]struct {
 }{
 EOF
 
-CRDS="MockConfig RouteRule IngressRule EgressRule DestinationPolicy HTTPAPISpec HTTPAPISpecBinding QuotaSpec QuotaSpecBinding"
+CRDS="MockConfig RouteRule V1alpha2RouteRule IngressRule EgressRule DestinationPolicy HTTPAPISpec HTTPAPISpecBinding QuotaSpec QuotaSpecBinding"
```

Comment:
I don't know much about the CRD internals to comment. This sounds strange though. I thought the idea behind doing v1alpha2 was to be able to have same CRD, but allow people to use different API versions?

Reply:
ah, yes. that would make more sense.